### PR TITLE
fix implied global `finish_shutdown`

### PR DESCRIPTION
**Pull request by vvo**
linked with #129

Seems to have been trigered by e21cf63

### CODE REVIEW

File: lib/SauceBrowser.js [hunk 1]
```diff
@@ -199,7 +199,7 @@ SauceBrowser.prototype.shutdown = function(err) {
 
     self.stopped = true;
 
-    finish_shutdown = function() {
```
Comment (vvo) [inline lib/SauceBrowser.js]: This cannot happen when using http://eslint.org/ (or $linter).

**Comment by rase-** [comment]
Wut. Good catch!

File: lib/SauceBrowser.js [hunk 2]
```diff
@@ -199,7 +199,7 @@ SauceBrowser.prototype.shutdown = function(err) {
...
+    self.finish_shutdown = function() {
```
Comment (rase-) [inline lib/SauceBrowser.js]: Hmm. I would've probably just made a var out of this. Not sure it feels right to put a function to `self` within a function.

Comment (vvo) [inline lib/SauceBrowser.js]: too quick fix, gonna check again

Comment (vvo) [inline lib/SauceBrowser.js]: @rase- fixed with dcfd8d09f36cf6e2b1be63ab2e999094b93bb770

**Comment by rase-** [comment]
Looks good! Thanks!

**Pull request merged**